### ISSUE TITLE
K8up user CLI Snapshots and Restores

### ISSUE THREAD

**Issue by Kidswiss** [k8up-io/k8up#643]
### Summary

**As** a user\
**I want** a CLI to handle snapshots and restores\
**So that** can easily restore data

### Context

Should be done after #122.

Once we can manage the snapshots via CRs, we should make it even easier to trigger restores with a given K8up CLI.

### Out of Scope

* backup
* schedules
* repositories

### Further links

* 

### Acceptance Criteria

```gherkin
Given a K8up snapshot
When executing `k8up snapshots list -n namespace`,
Then list snapshots for the namespace `namespace`
```
```gherkin
Given a K8up snapshot ID,
And a PVC `restore` in namespace `namespace`,
When executing `k8up restore --snapshot ID --target restore`
Then restore all files from the snapshot to the target PVC root
```
```gherkin
Given an exported kubeconfig
When running any `k8up restore` command
Then connect to the configured cluster
```
* Any CLI flag should also be set-able via ENV var

### Implementation Ideas

* Seems like there are already stubs prepared for a user cli: https://github.com/k8up-io/k8up/blob/master/cmd/k8up/main.go
* The `restore` subcommand will basically just create a templated `restore` CR, so the logic should be fairly simple
